lib/remix.js

```diff
@@ -13,7 +13,7 @@ function injectMetadata(html, metadata) {
     metaTags += '<meta name="data-remix-' + key + '" content="' + metadata[key] + '">\n';
   });
 
-  return html.replace(/<head>\n/, '$&' + metaTags);
+  return html.replace(/<head>\n?/, '$&' + metaTags);
```

Comment:
that sounds like a solid assumption. There are a lot of weird but legal HTML5 documents possible, but for the purposes of something like brimble assuming `<head>` seems sensible. It can also be forced from the thimble side, so that if people try to publish to our endpoint without a `<head>...</head>` block, that's an error. Even if HTML5 allows lots of different legal forms, our publish.wmo can be stricter about what it expects.